contracts/payment/CancellablePullPayment.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.11;
+
+
+import "./PullPayment.sol";
+
+
+/*
+ * @title CancellablePullPayment
+ * 
+ * @dev Inherits from PullPayment and allows the payment sender to cancel
+ * @dev a payment before it's withdrawn by the reciever.
+ * @dev Inherit from this contract and use asyncSend instead of send.
+ */
+
+contract CancellablePullPayment is PullPayment {
+
+  mapping(address => mapping(address => uint )) public paymentsByPayer;
```

Comment:
@ataddei I think you're right! What do you think about me splitting this up into a `CancellablePullPayment` contract and a `RefundablePullPayment` contract?